base_synchro/wizard/base_synchro.py

```diff
@@ -165,6 +181,52 @@ def get_id(self, object_id, id, action):
         return result
 
     @api.model
+    def get_existing_dst(self, pool_src, pool_dest, object_to_sync, res_id, action):
+
```

Comment:
Jay, I do not understand what do you mean with "We should write method before it is called.". Do you mean you need a _hook_?